### ISSUE TITLE
No able to retrieve recommendations

### ISSUE THREAD

**Issue by facundofarias** [Recombee/php-api-client#5]
Trace is:

```
[2017-09-04 13:38:26] local.nested.net.ERROR: Recombee\RecommApi\Exceptions\ResponseException: {"statusCode":404, "error":"Unknown key or database"}
 in /home/vagrant/Code/vendor/recombee/php-api-client/src/RecommApi/Client.php:167
Stack trace:
#0 /home/vagrant/Code/vendor/recombee/php-api-client/src/RecommApi/Client.php(155): Recombee\RecommApi\Client->checkErrors(Object(Requests_Response))
#1 /home/vagrant/Code/vendor/recombee/php-api-client/src/RecommApi/Client.php(90): Recombee\RecommApi\Client->post('http://rapi.rec...', 3, '{"count":5}')
```

**Comment by OndraFiedler** [comment]
I tried to reproduced the bug (both using my testing database and your database) and getting recommendations worked for me in both cases (using last released version `1.4.0` of the SDK).
What version of the SDK and what version of PHP do you use?

**Comment by facundofarias** [comment]
Hello @OndraFiedler!
I've done: `artisan config:clear`, and then it just started working. 

Thanks! 👍
